solidity/random-beacon/package.json

```diff
@@ -30,15 +30,16 @@
     "prepack": "tsc -p tsconfig.export.json && hardhat export-artifacts export/artifacts"
   },
   "dependencies": {
-    "@keep-network/sortition-pools": "1.2.0-dev.24",
+    "@keep-network/sortition-pools": "^2.0.0-pre.6",
```

Comment:
Shouldn't we use `"@keep-network/sortition-pools": "^2.0.0-dev.0",` as in `solidity/ecdsa/package.json`?

Reply:
We should use `pre`, as we discontinued using `dev` suffix for sortition-pools as per https://github.com/keep-network/sortition-pools/pull/149.

Reply:
OK, let's just remember to update it in `solidity/ecdsa` then.

Reply:
9fbb13386